stdlib/contrib/bonitoo-io/hex/hex.flux

```diff
@@ -0,0 +1,10 @@
+package hex
+
+builtin int : (v: A) => int
+builtin string : (v: A) => string
+builtin uint : (v: A) => uint
+builtin bytes : (v: A) => bytes
```

Comment:
Can the value for `v` be any type? or does it need to be a string? If it needs to be a string can you change these types to `builtin int : (v: stirng) => int`?

Reply:
Yes it can, it behaves the same way as standard `int()`, but a hex string value is expected.